joomla-projects/gsoc18_override_management
# Notification after update and TEST

Pull Request for Issue #15 

### Summary of Changes
An algorithm is implemented to find updated core files during an update ( Extension or Joomla update).


### Testing Instructions
Try to install the upgrade package of any extension. (Using package installation or direct server update or any core methods to upgrade )
And make sure that you have created the override of the extension.

Using -:

1. **Try any of these four sub-options.** 
![screenshot from 2018-06-16 15-53-13](https://user-images.githubusercontent.com/30978328/41497930-ab17bc16-717d-11e8-99ee-0f67182ec724.png)

2.  **Update manager**
![screenshot from 2018-06-16 15-56-47](https://user-images.githubusercontent.com/30978328/41497939-f7c4621c-717d-11e8-8b4f-5d546169d888.png)

3. **Joomla Update**
![screenshot from 2018-06-16 15-58-07](https://user-images.githubusercontent.com/30978328/41497942-15a7fef6-717e-11e8-8030-f2a18d114a4b.png)


### Expected result
It shows the notification that core files is changed during the update ( If really core files is changed then it shows the notification ).

**It shows this.**
![screenshot from 2018-06-16 16-00-48](https://user-images.githubusercontent.com/30978328/41497949-79c43378-717e-11e8-9903-203e46b2f8bb.png)


### Notes for TEST
I am storing result in <code>.txt</code> file. Checkout result file in `administrator` folder which contains the information about update.
